taraxa/state/dpos/tests/dpos_test.go

```diff
@@ -186,36 +193,53 @@ func TestRewardsAndCommission(t *testing.T) {
 
 	validator1_stats := rewards_stats.ValidatorStats{}
 	validator1_stats.UniqueTxsCount = 8
-	validator1_stats.ValidCertVote = false
+	validator1_stats.ValidCertVote = true
 	initValidatorTxsStats(validator1_addr, &fees_rewards, txFee, validator1_stats.UniqueTxsCount)
 	tmp_rewards_stats.ValidatorsStats[validator1_addr] = validator1_stats
 
 	validator2_stats := rewards_stats.ValidatorStats{}
 	validator2_stats.UniqueTxsCount = 32
-	validator2_stats.ValidCertVote = false
+	validator2_stats.ValidCertVote = true
 	initValidatorTxsStats(validator2_addr, &fees_rewards, txFee, validator2_stats.UniqueTxsCount)
 	tmp_rewards_stats.ValidatorsStats[validator2_addr] = validator2_stats
 
 	tmp_rewards_stats.TotalUniqueTxsCount = validator1_stats.UniqueTxsCount + validator2_stats.UniqueTxsCount
-	tmp_rewards_stats.TotalUniqueVotesCount = 0
+	tmp_rewards_stats.TotalVotesCount = 2
+	tmp_rewards_stats.BonusVotesCount = 1
```

Comment:
I think it would e nice to have actually a valid scenario in tests. We could add 2 more validators and it would be valid then to have total_votes = 4, out of which 1 is bonus vote